src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1674,6 +1674,7 @@ class DocBaseViewer extends BaseViewer {
 
     handleAnnotationColorChange(color) {
         this.annotationModule.setColor(color);
+        this.annotator.emit(ANNOTATOR_EVENT.setStroke, { color });
```

Comment:
This seems very specific. Does this model scale well as we add more options? Why is there an object wrapper and how/where is its type defined?

Reply:
Since this is under handle color change function, I changed the event to `setColor`. Also changed the payload to a color string.